deploy/charts/cert-manager/templates/controller-config.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.config -}}
-{{- required ".Values.config.apiVersion must be set !" .Values.config.apiVersion -}}
-{{- required ".Values.config.kind must be set !" .Values.config.kind -}}
+{{- $_ := .Values.config.apiVersion | required ".Values.config.apiVersion must be set !" -}}
+{{- $_ := .Values.config.kind | required ".Values.config.kind must be set !" -}}
```

Comment:
Could you apply the same fix to https://github.com/cert-manager/cert-manager/blob/master/deploy/charts/cert-manager/templates/cainjector-config.yaml?

Reply:
Hey @inteon, no problem :) 

When I did this improvement I actually tested it locally and the issue didn't appear. I applied the same kind of process to one of my Chart a couple of days ago and the values got templated. 

So I tested it locally again on the cert-manager Chart yesterday but the values are still not templated. No clue why it behaves differently between my chart and cert-manager's though (or maybe I simply didn't notice properly) :/

Yes I will ;)

Reply:
@inteon Just pushed the same fix ;)

Reply:
@inteon Just pushed :)